docs/tests/integration/test_vulnerability_detector/index.md

```diff
@@ -2,21 +2,21 @@
 
 Wazuh is able to detect vulnerabilities in the applications installed in agents using the `Vulnerability Detector`
 module. This software audit is performed through the integration of vulnerability feeds indexed by `Canonical`,
-`Debian`, `RedHat`, and the `National Vulnerability Database`.
+`Debian`, `RedHat`, `Amazon Linux` and the `National Vulnerability Database`.
 
 This directory includes all the integration tests developed to test the correct functioning of this module.
 
 These tests can be classified in these categories:
 
-- **[test_feeds](test_feeds#test-feeds)**: Tests that check the behavior of Vulnerability Detector when a feed with an
+- **[test_feeds](test_feeds/#test-feeds)**: Tests that check the behavior of Vulnerability Detector when a feed with an
 unexpected content/type is imported, feeds downloads...
 
-- **[test_general_settings](test_general_settings#test-general-settings)**: Tests that check basic configuration of the
+- **[test_general_settings](test_general_settings/#test-general-settings)**: Tests that check basic configuration of the
 Vulnerability Detector in the `ossec.conf`.
 
-- **[test_providers](test_providers#test-providers)**: Tests that check the providers configuration in the `ossec.conf`.
+- **[test_providers](test_providers/#test-providers)**: Tests that check the providers configuration in the `ossec.conf`.
```

Comment:
In the future, to be more specific in 5.0, `ossec.conf` will become `wazuh.conf`. For that, we will need to apply a lot of changes if we add `ossec.conf` in the documentation. 

Maybe you can add something like:
``` 
Tests that check the provider's configuration in the main configuration file.
```

If you agree, you should modify it in the new documentation created.

Reply:
As we agreed in the daily, we will keep the same file name.